app/containers/Main/companyRoutes.js

```diff
@@ -64,6 +64,7 @@ const CompanyRoutes = () => (
   <Switch>
     <Route exact path="/" component={PublicMain} />
     <Route exact path="/apply" component={PublicJobApplication} />
+    <Route exact path="/apply/:positionId" component={PublicCompanyPositionDetail} />
```

Comment:
Wait you changed the position detail to `/apply` instead of `/jobs`? This is going to break all our existing links.